neurodamus/target_manager.py

```diff
@@ -600,118 +599,31 @@ def generate_subtargets(self, n_parts):
         return [[targets[cycle_i] for targets in new_targets.values()]
                     for cycle_i in range(n_parts)]
 
+    def set_offset(self, *_):
+        pass  # Only hoc targets require manually setting offsets
```

Comment:
Do we still need to `set_offset`?

Reply:
you are right, this was called from few places but I guess was only for hoc targets, will remove that also